disttask/framework/scheduler/interface.go

```diff
@@ -0,0 +1,65 @@
+// Copyright 2023 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package scheduler
+
+import (
+	"context"
+
+	"github.com/pingcap/tidb/disttask/framework/proto"
+)
+
+// TaskTable defines the interface to access task table.
+type TaskTable interface {
```

Comment:
Is it used for mock? Could these interfaces be moved to storage package?

Reply:
Agree, I can move it to storage pkg in later pr.

Reply:
Please add a TODO.